Source/BrewInfo.spoon/init.lua

```diff
@@ -0,0 +1,137 @@
+--- === BrewInfo ===
+---
+--- Display pop-up with Homebrew Formula info, or open their URL
+---
+--- Download: [https://github.com/Hammerspoon/Spoons/raw/master/Spoons/BreInfo.spoon.zip](https://github.com/Hammerspoon/Spoons/raw/master/Spoons/BrewInfo.spoon.zip)
```

Comment:
Ah, good catch, fixed.

BTW since then I've been working on a "newspoon" script which gives you a skeleton template with some information pre-filled. I'll share it when it's in a more mature shape.